apps/docs/components/blocks/MegaMenu.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: Mega menu is a type of website navigation menu that typically displays a list of links and subcategories in a larger, more complex format than a traditional drop-down or fly-out menu. 
+---
+# MegaMenu
+
+{{ $frontmatter.description }}
+
+ ## Accessibility notes
+
+MegaMenu follows the accessibility guidelines for menus and menubars by WCAG [Read more](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/).
+
+ ## Opening on click
```

Comment:
Do you think this name is accurate? I know it's a part of that block, but in general it's more like a "Basic MegaMenu". Just saying. 😄

Reply:
WIth that please also rename function name in react component 
`export default function OpenOnClick() {`